crates/proto/src/rr/domain/name.rs

```diff
@@ -665,17 +665,6 @@ impl Name {
         Ok(name)
     }
 
-    /// Emits the canonical version of the name to the encoder.
-    ///
-    /// In canonical form, there will be no pointers written to the encoder (i.e. no compression).
-    pub fn emit_as_canonical(
```

Comment:
This should be removed with the last user.

Reply:
The last use is the call in `<Name as BinEncodable>::emit()` that we are inlining now.